changelogs/fragments/rename_key_to_key_id_junos_ntp_global.yaml

```
@@ -0,0 +1,3 @@
+---
+minor_changes:
```

The inclusion of this module is already added as a minor_change.
```suggestion
trivial:
```